src/cd-device-db.c

```diff
@@ -79,7 +79,7 @@ cd_device_db_load (CdDeviceDb *ddb,
 
 	/* check devices */
 	rc = sqlite3_exec (priv->db, "SELECT * FROM devices LIMIT 1",
-			   NULL, NULL, &error_msg);
+			   NULL, NULL, NULL);
 	if (rc != SQLITE_OK) {
 		g_debug ("CdDeviceDb: creating table to repair: %s", error_msg);
 		sqlite3_free (error_msg);
```

Comment:
I think we need the error_msg here...

Reply:
It seems that error_msg is not used. If we use it, we should free it.

Reply:
It really **is** used in that first case.

Reply:
Sorry for that. I submitted the issue a long time ago and I missed that the error_msg is needed here now. I will leave this error_msg and re-PR.